### DISCUSSION TITLE
How to create 3D model without AR using arsceneview

### DISCUSSION THREAD

**Discussion by mecherguikhaoula1234** [sceneview/sceneview#344] [Q&A]
It is mlentionned in the documentation that this dependency 'io.github.sceneview:arsceneview:1.1.0' is for 3D and ARCore.
i have two questions:
1/ is it possible to display object in 3D mode with the component:
<io.github.sceneview.ar.ArSceneView android:id="@+id/sceneView" android:layout_width="match_parent"
android:layout_height="match_parent" /> . if yes how to use it in 3d mode ?
2/ if it is not possible for my first question: how to display object in 3D mode with the compnent :
<io.github.sceneview.SceneView android:id="@+id/sceneView" android:layout_width="match_parent"
android:layout_height="match_parent" /> and i want the one that exists in the dependency that i am using and mentioned above .
Thank you in adavance

**Comment by ThomasGorisse** [comment]
You have two choices:
- Use an `ARSceneView` and a `SceneView` sharing the same engine and nodes if you need them separated in the layout.
- Use a single instance `ARSceneView`, disable the camera stream, change the `ARCameraNode` to a `CameraNode` and add a Skybox

Note that, in first case,  you can also share the `CameraNode` to have the same point of view

**Comment by ThomasGorisse** [comment]
It may also depend on your need to share the main and indirect light. In case of light estimation enabled you can decide on letting it estimated or fixed.

**Comment by mecherguikhaoula1234** [comment]
> Use a single instance ARSceneView, disable the camera stream, change the ARCameraNode to a CameraNode and add a Skybox

Thank you for your return.
Is it possible to give me an example on how to do the second choice : 
I want to know how to **disable the camera stream, change the ARCameraNode to a CameraNode**

**Comment by ThomasGorisse** [comment]
Use the `sceneview` artifact (not `arsceneview`) for 3D without AR:

```kotlin
implementation("io.github.sceneview:sceneview:4.0.0-rc.1")

SceneView {
    val model = rememberModelInstance(modelLoader, "models/mymodel.glb")
    if (model != null) {
        ModelNode(modelInstance = model, scaleToUnits = 1.0f)
    }
}
```

See also #246 for more details.
